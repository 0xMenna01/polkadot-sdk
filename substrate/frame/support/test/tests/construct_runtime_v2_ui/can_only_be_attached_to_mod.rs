#[frame_support::construct_runtime_v2]
fn construct_runtime() {}

fn main() {}
