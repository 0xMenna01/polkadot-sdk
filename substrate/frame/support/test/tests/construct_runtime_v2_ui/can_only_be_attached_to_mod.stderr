error: expected `mod`
  --> tests/construct_runtime_v2_ui/can_only_be_attached_to_mod.rs:19:1
   |
19 | fn construct_runtime() {}
   | ^^
